resources/qgis-base-metadata.xsd

```diff
@@ -102,6 +107,25 @@
 			</xs:annotation>
 		</xs:attribute>
 	</xs:complexType>
+	<xs:complexType name="datesType">
+		<xs:sequence>
+			<xs:element name="date" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>A dates associated with a resource.</xs:documentation>
+				</xs:annotation>
+				<xs:attribute name="type" type="xs:string">
+					<xs:annotation>
+						<xs:documentation>Date type, valid values are Created, Published, Revised, Superseded.</xs:documentation>
```

Comment:
Could the `type` attribute be an enumeration?

Reply:
Good point, done!